packages/core/test/odata/payload-value-converter.spec.ts

```diff
@@ -388,3 +433,19 @@ describe('edm to moment and back', () => {
     );
   });
 });
+
+function checkInfinityCasesToEdm(edmType: EdmType) {
+  expect(tsToEdm(Number.POSITIVE_INFINITY, edmType)).toBe('INF');
+  expect(tsToEdm(Number.NEGATIVE_INFINITY, edmType)).toBe('-INF');
+  expect(tsToEdm(Number.NaN, edmType)).toBe('NaN');
+}
+
+function checkInfinityCasesToTs(edmType: EdmType) {
+  expect(edmToTs('INF', edmType)).toBe(Number.POSITIVE_INFINITY);
+  expect(edmToTs('-INF', edmType)).toBe(Number.NEGATIVE_INFINITY);
+  expect(edmToTs('NaN', edmType)).toBe(Number.NaN);
+}
+
+function checkInfinityCasesRoundTrip(edmType: EdmType) {
+  ['NaN'].forEach(s => expect(tsToEdm(edmToTs(s, edmType), edmType)).toBe(s));
```

Comment:
Why a list?

Reply:
ah had a failing test for NaN and forgot to put the other back. Thanks